chainer/_runtime_info.py

```diff
@@ -51,13 +51,39 @@ def __str__(self):
         return s.getvalue()
 
 
-def get_runtime_info():
+def _get_runtime_info():
     return _RuntimeInfo()
 
 
 def print_runtime_info(out=None):
+    """Shows Chainer runtime information.
+
+    Runtime information includes:
+
+    - OS platform
+
+    - Chainer version
+
+    - ChainerX version
+
+    - NumPy version
+
+    - CuPy version
+
+      a) CUDA information
```

Comment:
How about just using bullets (`-`)?
I feel it's more natural as this list is unordered.

Reply:
I got it. :+1: